### PR TITLE
#228 add empty state for sections on the profile page

### PR DESCRIPTION
**Pull request by rachelchenrc**
[Trello card](https://trello.com/c/1JxhRQsw/228-as-a-user-i-want-to-see-an-empty-state-for-any-unpopulated-sections-on-the-profile)
[Live env](https://nj-career-network-dev4.firebaseapp.com/profile)

- Add empty-state profile item cards 
- Pull user phone number from assessment answer and display in user profile card

Screen Shot 2020-05-11 at 10 14 43 AM

### CODE REVIEW

File: components/profile/Profile.js [hunk 1]
```diff
@@ -1,27 +1,131 @@
...
+  },
+];
+
+const PHONE_QUESTION_ID = 'recDkvWeqvMxahFT2';
+
+function Profile({ allQuestionResponses }) {
   const classes = useStyles();
+  const { user } = useAuth();
+  const questionResponse = allQuestionResponses.find(doc => doc.id === PHONE_QUESTION_ID);
+  const phoneNumber = questionResponse && questionResponse.data().value;
```
Comment (hannahtwobulls) [inline components/profile/Profile.js:95]: Is the `phoneNumber` accessible via the `user`? (i.e. do we have to retrieve it from the assessment?)

File: components/profile/UserProfileCard.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <CardContent className={classes.cardContent}>
+          <Grid container alignItems="center" spacing={1} className={classes.cardHeader}>
+            <Grid item>
+              <Avatar src={photoURL} alt={displayName} className={classes.avatar}>
+                <PersonIcon />
+              </Avatar>
+            </Grid>
+            <Grid item>
+              <Typography variant="h6" gutterBottom>
+                {user.displayName}
```
Comment (hannahtwobulls) [inline components/profile/UserProfileCard.js:52]: This is already destructured!

File: components/profile/Profile.js (hunk 1 above)
Comment (rachelchenrc) [reply components/profile/Profile.js:95]: the phoneNumber under user authProfile is generated by the auth provider,  and currently, I didn't see it get populated.

File: components/profile/UserProfileCard.js (hunk 2 above)
Comment (rachelchenrc) [reply components/profile/UserProfileCard.js:52]: thanks for catching it!

**Comment by rachelchenrc** [comment]
Rebased.

**Pull request merged**